cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportContext.java

```diff
@@ -53,12 +53,41 @@ public class CgmesExportContext {
     private final BiMap<String, String> regionsIdsByRegionName = HashBiMap.create();
     private final BiMap<String, String> subRegionsIdsBySubRegionName = HashBiMap.create();
 
+    // Update dependencies in a way that:
+    // SV.dependentOn TP
+    // SV.dependentOn SSH
+    // TP.dependentOn EQ
+    // SSH.dependentOn EQ
+    public void updateDependencies() {
+        String eqModelId = getEqModelDescription().getId();
+        if (eqModelId != null) {
+            List<String> eqDependency = List.of(eqModelId);
+            getTpModelDescription()
+                    .clearDependencies()
+                    .addDependencies(eqDependency);
+            getSshModelDescription()
+                    .clearDependencies()
+                    .addDependencies(eqDependency);
+            getSvModelDescription().clearDependencies();
+            String tpModelId = getTpModelDescription().getId();
+            if (tpModelId != null) {
+                getSvModelDescription().addDependencies(List.of(tpModelId));
```

Comment:
We can add a `addDependency` method

Reply:
Right. The method was already present in the `ModelDescription` class